### DISCUSSION TITLE
ADD COMPATIBILITY WITH NEW LIBRARIES 🙏

### DISCUSSION THREAD

**Discussion by dotaxel** [CodenameCrew/CodenameEngine#252] [Ideas]
add compatibility with the lastests libs and lastest haxe pleaseee

**Comment by charlesisfeline** [comment]
they'll probably do it in v1.1.0, im sure of it



edit: they did it, but rn its not fully supported in 4.3.x but hopefully in the next many updates they'll probably move to 4.3.x
